docs/sources/operators-guide/get-started/_index.md

```diff
@@ -9,112 +9,114 @@ weight: 10
 
 # Get started with Grafana Phlare
 
-There are two different options for getting started with Grafana Phlare:
+Choose one of the following options to get started with Grafana Phlare:
 
-- The written tutorial provides a series of imperative commands to start a single Phlare process.
-- The visual tutorial (in the form of a video) uses [`docker-compose`](https://github.com/grafana/phlare/tree/main/tools/docker-compose) to declaratively deploy Phlare and Grafana.
+- The **written tutorial** below provides a series of imperative commands to start a single Phlare process, or [monolith]({{< relref "../architecture/deployment-modes/index.md#monolithic-mode" >}}), which is designed for users getting started with the project. 
 
-{{< vimeo todo >}}
+- The following **video tutorial** uses [`docker-compose`](https://github.com/grafana/phlare/tree/main/tools/docker-compose) to declaratively deploy Phlare and Grafana.
 
-<br/>
+  {{< vimeo todo >}}
```

Comment:
Not too sure if that video is ready somewhere?, but we shouldn't block merging this

cc @cyriltovena

Reply:
Fixed